src/__tests__/home.test.js

```diff
@@ -0,0 +1,20 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Home from '../components/home';
+import store from '../redux/store';
+
+describe('Home', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <BrowserRouter>
+            <Home />
+          </BrowserRouter>
+        </Provider>,
+      )
```

Comment:
- [ ] As requested by the previous reviewer please make sure to test the pure functions in your Redux code.